src/features/TxStatus/TxStatus.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+import { simpleRender } from 'test-utils';
+
+import { NetworkContext } from '@services';
+import { translateRaw } from '@translations';
+
+import TxStatus from './TxStatus';
+
+const TX_HASH = '0xb324e6630491f89aff0e8e30228741cbccc7ddfdb94c91eedc02141b1acc4df7';
+
+/* Test components */
+describe('TxStatus', () => {
+  const component = (path?: string) => (
+    <MemoryRouter initialEntries={path ? [path] : undefined}>
+      <NetworkContext.Provider value={{ getNetworkById: jest.fn(), networks: [] } as any}>
+        <TxStatus />
+      </NetworkContext.Provider>
+    </MemoryRouter>
+  );
+
+  const renderComponent = (pathToLoad?: string) => {
+    return simpleRender(component(pathToLoad));
+  };
+
+  test('Can render', () => {
+    const { getByText } = renderComponent();
+    const selector = translateRaw('TX_STATUS');
+    expect(getByText(selector)).toBeInTheDocument();
+  });
+
+  test('Can get hash from query string', () => {
+    const { getByText, getByDisplayValue } = renderComponent(`/tx-status?hash=${TX_HASH}`);
+    const selector = translateRaw('TX_STATUS');
+    expect(getByText(selector)).toBeInTheDocument();
+    expect(getByDisplayValue(TX_HASH)).toBeInTheDocument();
+  });
```

Comment:
Are there only 2 states for the component?
1. empty without query string.
2. pre-filled with query string.

- Please be more explicit in the description with a "when X then Y" formula.
- Mock the network request to avoid slowing down the test suite.